signalfx-org-metrics/metrics.yaml

```diff
@@ -1111,12 +1135,22 @@ sf.org.numLimitedMetricTimeSeriesCreateCalls:
   metric_type: counter
   title: sf.org.numLimitedMetricTimeSeriesCreateCalls
 
+sf.org.numLimitedMetricTimeSeriesCreateCallsByCategoryType:
+  brief: Number of MTS not created because your account reached a category limit (or subscription limit).
+  description: |
+    Number of metric time series (MTS) not created because your account reached a category limit (or subscription limit).
+
+        * Dimension(s):  `categoryType`, `orgId`
+        * Data resolution: 35 seconds
```

Comment:
Not sure 35 seconds is correct. That would be a weird resolution if it's true.

Reply:
Resolution here should be same as any other org metric.